dtool/Config.cmake

```diff
@@ -36,15 +36,15 @@ endif()
 
 # Define the type of build we are setting up.
 if(IS_MULTICONFIG)
-  set(CMAKE_CONFIGURATION_TYPES Release RelWithDebInfo Debug MinSizeRel Distribution)
+  set(CMAKE_CONFIGURATION_TYPES Release RelWithDebInfo Debug MinSizeRel Standard)
 else()
   # CMAKE_BUILD_TYPE can't just be set using the usual set(CACHE) method since
   # it's an empty string by default.
   if(NOT CMAKE_BUILD_TYPE)
     set(CMAKE_BUILD_TYPE RelWithDebInfo)
   endif()
   set_property(CACHE CMAKE_BUILD_TYPE PROPERTY STRINGS
-    Release RelWithDebInfo Debug MinSizeRel Distribution)
+    Release RelWithDebInfo Debug MinSizeRel Standard)
```

Comment:
Let's put Standard first in the list, and make it the default as opposed to RelWithDebInfo.  Standard corresponds to what is currently the makepanda default, and what we want the majority of developers building Panda from source for the first time will want to use.